### ISSUE TITLE
false positive: assert.True can not be used with types defined from bool

### ISSUE THREAD

**Issue by scop** [Antonboom/testifylint#65]
testifylint (the version included with golangci-lint 1.56.1) suggests to replace
```go
assert.EqualValues(t, true, something)
```
...where the type of `something` is defined from `bool`, like `type MyBool bool`. Following the suggestion, with `assert.True`, would not compile.

**Comment by Antonboom** [comment]
@scop, hi!

thank you for issue

I propose to introduce `--bool-compare.ignore-named-types`, default `false` and fix will be 
```
var b MyBool
assert.Equal(t, false, b)  ->  assert.False(t, bool(b))
```

How do you find such option?
